code/game/gamemodes/colonialmarines/colonialmarines.dm

```diff
@@ -65,6 +64,7 @@
 /datum/game_mode/colonialmarines/post_setup()
 	initialize_post_queen_list()
 	initialize_post_xenomorph_list()
+	initialize_starting_survivor_list()
```

Comment:
Curious change, the gamemode code is ran before the jobs code, won't this mean there won't be any minds to pick from?